### DISCUSSION TITLE
How to prevent Carousel indicator buttons from scrolling entire page?

### DISCUSSION THREAD

**Discussion by loboroberto** [saadeghi/daisyui#1248] [General]
In the Carousel component page, the demo for an example with indicator buttons has the same behavior as when Ive implemented into my own page.  Using href=#x" causes the browser to scroll to that element... In addition to the carousel sliding.  How can I prevent this behavior and only see the carousel slide?

**Comment by saadeghi** [comment]
See this: https://github.com/saadeghi/daisyui/issues/413

Let me know if you have any questions.
